awx/main/utils/execution_environments.py

```diff
@@ -6,10 +6,14 @@
 from awx.main.models.execution_environments import ExecutionEnvironment
 
 
+def get_control_plane_execution_environment():
+    return ExecutionEnvironment.objects.filter(organization=None, managed_by_tower=True).first()
```

Comment:
Perhaps it's already established as a TODO, but since the pre-load EEs are concretely going to be a list with >1 elements, we need a selection criteria other than `.first()`

Reply:
Based on the ordering criteria I imposed on the EE model way back when `.first()` translates to 'most recent', which I think is the correct priority we should give to installed EEs.

Reply:
intended use (near-term) is that someone passes a list of 4 EEs to create in the installer

the `awx-manage` command loops over those and creates them

what you're saying would indicate that jobs will use the _final_ EE in the list as the default. This is high-value information, so much so that I will @Spredzy since he may be implementing the installer side of this.

Reply:
Yes, though the idea originally was that newer EEs installed when updating would be preferentially used over prior ones.